test/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ SET(TEST_SOURCES
   ${CMAKE_CURRENT_LIST_DIR}/main.cxx
   ${CMAKE_CURRENT_LIST_DIR}/Example.cxx
   ${CMAKE_CURRENT_LIST_DIR}/engine/ResourcesManager.cxx  
+  ${CMAKE_CURRENT_LIST_DIR}/engine/AudioMixer.cxx  
```

Comment:
There is a "Enable SDL2 Mixer" CMake variable. You should check for it too, or else tests will fail if that is disabled.